java/ql/lib/semmle/code/java/frameworks/Servlets.qll

```diff
@@ -377,3 +377,26 @@ class RequestDispatchMethod extends Method {
     this.hasName(["forward", "include"])
   }
 }
+
+/**
+ * The interface `javax.servlet.ServletContext`.
+ */
+library class ServletContext extends RefType {
```

Comment:
Don't use `library`, it's deprecated.

```suggestion
class ServletContext extends RefType {
```

Reply:
I've made requested change. Please also note the class `ServletContext` itself is not deprecated, which is still widely and actively used in Java EE applications as a main method to interact with the servlet container, although several methods related to `getServlet` and `log` were deprecated. Please let me know if we want to keep the `library` modifier. Thanks.

Reply:
What I meant is that the `library` annotation is deprecated in CodeQL. The `ServletContext` can still be used normally.

Reply:
Thanks @atorralba for the clarification.